dask/array/routines.py

```diff
@@ -1843,6 +1843,50 @@ def piecewise(x, condlist, funclist, *args, **kw):
     )
 
 
+def _select(*args, **kwargs):
+    split_at = len(args) // 2
+    condlist = args[:split_at]
+    choicelist = args[split_at:]
+    return np.select(condlist, choicelist, **kwargs)
+
+
+@derived_from(np)
+def select(condlist, choicelist, default=0):
+    # Making the same checks that np.select
+    # Check the size of condlist and choicelist are the same, or abort.
+    if len(condlist) != len(choicelist):
+        raise ValueError("list of cases must be same length as list of conditions")
```

Comment:
I like that you took the same error messages as numpy raises :)

Reply:
Yeah, there is a problem in doing that? I saw several disclaimers about numpy, and its license, I didn't know if I had to add something more.

Reply:
I think this is perfect!